presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -252,6 +252,8 @@
     // TODO: Native execution related session properties that are temporarily put here. They will be relocated in the future.
     public static final String NATIVE_SIMPLIFIED_EXPRESSION_EVALUATION_ENABLED = "simplified_expression_evaluation_enabled";
     public static final String NATIVE_AGGREGATION_SPILL_MEMORY_THRESHOLD = "aggregation_spill_memory_threshold";
+    public static final String NATIVE_JOIN_SPILL_MEMORY_THRESHOLD = "join_spill_memory_threshold";
```

Comment:
should the property start with native if it is only supported by native?

Reply:
> should the property start with native if it is only supported by native?

I thought about it, but then on Native Code side (Velox) it should have "native_" prefix too, which would look weird.
Unless we do some sort of name manipulation in cpp layer.
Need to think about it.

Reply:
It is ok for now, as other properties has the similar naming.